speakeasy/winenv/api/usermode/shell32.py

```diff
@@ -96,6 +96,23 @@ def ShellExecute(self, emu, argv, ctx={}):
 
         return 33
 
+    @apihook('ShellExecuteEx', argc=1)
+    def ShellExecuteEx(self, emu, argv, ctx={}):
+        '''
+        BOOL ShellExecuteExA(
+            [in, out] SHELLEXECUTEINFOA *pExecInfo
+        );
+        '''
+        lpShellExecuteInfo, = argv
+
+        sei = shell32_defs.SHELLEXECUTEINFOA(emu.get_ptr_size())
+        sei_struct = self.mem_cast(sei, lpShellExecuteInfo)
+
+        self.ShellExecute(emu,[ 0, sei_struct.lpVerb, sei_struct.lpFile,
+                          sei_struct.lpParameters, sei_struct.lpDirectory, 0], ctx)
```

Comment:
```suggestion
        self.ShellExecute(
            emu,
            [0, sei_struct.lpVerb, sei_struct.lpFile, sei_struct.lpParameters, sei_struct.lpDirectory, 0], 
            ctx
        )
```

simply formatting.

i like this way of sharing the implementation, at least while the two hooks support the same things.